libs/langchain/langchain/vectorstores/pathway.py

```diff
@@ -0,0 +1,6 @@
+from langchain_community.vectorstores.pathway import (
```

Comment:
as mentioned, lets not import this into langchain - langchain should remain unchanged

Reply:
Sorry, we did it mimicking how the other vectorstores were integrated.

However, if we remove this import, how should we proceed with the self-query retriever? Similarly to other vectorstores, we have extended the mapping of stores to translators at https://github.com/langchain-ai/langchain/pull/14859/files#diff-d9f3d054efd12eda9d8d1022d9d7023eb8579c3f4842a051b5f459927f4f1426R74 but this also changes core LangChain. What do you suggest @hwchase17?

Reply:
@hwchase17 I've noticed that the other vectorstores are now imported from langchain_community - I've changed the PR accordingly.